app/views/shared/_post_actions.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= link_to post, class: "btn btn-warning" do %>
+<%= link_to get_edit_path(post), class: "btn btn-warning" do %>
```

Comment:
what about this? 

```
link_to [:edit, post]
```

Reply:
I knew there was a better way, I've been doing this tons of times.... thank you for refreshing my memory @markets :heart_eyes: 

To pay for my sins I also created https://github.com/coopdevs/timeoverflow/pull/412